Tools/TestWebKitAPI/Tests/WebKitGLib/TestWebKitWebView.cpp

```diff
@@ -1611,6 +1611,30 @@ static void testWebViewBackgroundColor(WebViewTest* test, gconstpointer)
     // MiniBrowser --bg-color="<color-value>" for manually testing this API.
 }
 
+#if PLATFORM(WPE)
+static void testWebViewThemeColor(WebViewTest* test, gconstpointer)
+{
+    WebKitColor rgba;
+    webkit_web_view_get_theme_color(test->m_webView.get(), &rgba);
+
+    g_assert_cmpfloat(rgba.red, ==, 0);
+    g_assert_cmpfloat(rgba.green, ==, 0);
+    g_assert_cmpfloat(rgba.blue, ==, 0);
+    g_assert_cmpfloat(rgba.alpha, ==, 0);
+
+    test->loadHtml("<html style='width: 325px; height: 615px'><meta name='theme-color' content='#FF0000' /></html>", nullptr);
+    test->waitUntilLoadFinished();
+    test->wait(1);
```

Comment:
I don't think we should wait 1 second if it can be avoided, could we just monitor the property by connecting to the notify signal and start the run loop until the signal is emitted?